app/uk/gov/hmrc/nisp/views/includes/helpers/error_overview.scala.html

```diff
@@ -0,0 +1,16 @@
+@(form: Form[_])
+
+@if(form.hasErrors) {
+        <div id="error-summary-display" class="flash error-summary error-summary--show " role="group" aria-labelledby="error-summary-heading" tabindex="-1">
+            <h2 id="error-summary-heading" class="h3-heading">@Messages("nisp.validation.summary.heading")</h2>
+            <ul class="js-error-summary-messages">
+                @form.errors.map { error =>
+                <li role="tooltip">
+                    <a href='#@error.key' id='@error.key-error-summary' data-focuses='@error.key'>
+                        @if(error.key=="email") {@Messages("nisp.validation.summary.email") }else{ @Messages(error.message, error.args : _*)}
```

Comment:
This "nisp.validation.summary.error.max" is for the 2nd error advice, play-ui doesn't work with args for that reason I implemented error_overview and that was the original aim of this ticket. I can't see the point to change all the code.